templates/store/publisher-details.html

```diff
@@ -0,0 +1,212 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}{% endblock %}
+
+{# TODO: title, meta, etc #}
+{% block meta_title %}{{ publisher }} in the Snap Store{% endblock %}
+{% block meta_description %}Install {{ publisher }} software for Linux in the Snap Store{% endblock %}
+
+{% if icon_url %}
+  {% block meta_image %}
+    {{ icon_url }}
+  {% endblock %}
+{% endif %}
+
+
+{% block content %}
+  <div class="p-strip--dark is-shallow" style="background: #262626">
+    <div class="row">
+      <div class="p-snap-heading">
+        {% if icon_url %}
+          <img class="p-snap-heading__icon" src="{{ icon_url }}" alt="{{ publisher }} snap" />
+        {% else %}
+          <img class="p-snap-heading__icon" src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" alt="" />
+        {% endif %}
+        <div class="p-snap-heading__title">
+          <h1 class="p-heading--two p-snap-heading__name">{{ publisher }}</h1>
+          <p class="p-snap-heading__publisher">
+            ({{ username }})
+            {% if developer_validation and developer_validation == VERIFIED_PUBLISHER %}
+            <span class="p-verified p-tooltip p-tooltip--top-center" aria-describedby="{{ username }}-tooltip">
+              <img src="https://assets.ubuntu.com/v1/75654c90-rosette.svg" />
+              <span class="p-tooltip__message u-align--center" role="tooltip" id="{{ username }}-tooltip">Verified account</span>
+            </span>
+            {% endif %}
+          </p>
+        </div>
+
+        <div class="p-snap-install-buttons">
+          {% if website %}
+            <a href="{{ website }}" class="p-button--positive p-link--external">Visit publisher website</a>
+          {% endif %}
+          {% if contact %}
+            <a href="{{ contact }}" class="p-button--neutral p-link--external">Contact publisher</a>
+          {% endif %}
+        </div>
+
+      </div>
+    </div>
+  </div>
+
+  {% if banner_url %}
+  <section
+    class="p-strip--image is-deep"
+    style="min-height: 400px; background-image:url('{{ banner_url }}'); background-position: center;">
```

Comment:
can we put these in a pattern too (apart from `background-image` obvs?

Reply:
👍